pages/map.vue

```diff
@@ -83,12 +83,13 @@
         <button @click="clearSelectedOblasts">Clear</button>
       </div>
 
+      <!-- sample initial scenario -->
       <DataTable
-        :data-by-grain-type="originalDataByGrainType"
         :sorted-data-by-grain-type="sortedDataByGrainType"
-        :total-harvested-area="0"
-        :total-yield="0"
-        :total-volume="0"
+        :scenario="{
+          // sample scenario sets the first oblast to 50%
+          '7260': 0.5
```

Comment:
what is 7260?

Reply:
Also- we are not using this `pages/map.vue` implementation of the map anymore, it's been component-ized out to https://github.com/earthrise-media/story-ukraine-grain/blob/main/components/UkraineOblastMap.vue

So if this is important- and not just artifacts from testing/prototyping- I would make these changes there instead